python/tvm/relax/frontend/nn/core.py

```diff
@@ -67,6 +67,8 @@
 
 
 _DEFAULT_DTYPE = "float32"
+# If a shape is specified as a string `name`, we create a `tvm.tir.Var(name, "int64")`
+ShapeVar = Union[int, str, tir.PrimExpr]
```

Comment:
Would `ShapeInputType` be better, with a more detailed documentation above it? I am worried that if more `nn.Module` are going to have dynamic shapes in `python/tvm/relax/frontend/nn/modules.py`, `Union[int, str, tir.PrimExpr]` would be repeated in too many places... Right now it is only `Embedding` and `Linear`.

Reply:
I see, given this list do not grow, i still think being explicit would be better, since it also helps the readers to learn about what can be passed in directly through signature

Reply:
That makes sense, just fixed; thank you!